.vib/dokuwiki/cypress/cypress/integration/dokuwiki_spec.js

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login and logout', () => {
+  cy.visit('/');
+  cy.contains(Cypress.env('dokuwikiFullName'));
+  cy.login();
+  cy.get('.error').should('not.exist');
+  cy.contains('Log Out');
+});
+
+it('allows the user to update profile', () => {
+  cy.login();
+  cy.visit('/start?do=profile');
+  cy.fixture('admins').then((admin) => {
+    cy.get('input[name="email"]')
+      .clear()
+      .type(`${admin.newAdmin.email}.${random}`);
+  });
+  cy.get('input[name="oldpass"]').first().type(Cypress.env('password'));
```

Comment:
There are 2 password fields here with the same ID.
In this case we **always** want the first one. 

![Proper field](https://user-images.githubusercontent.com/47592680/166456277-5639b8ef-cc19-485f-8c67-8875922dc49f.png)

Reply:
All set then!